### DISCUSSION TITLE
Code of Conduct Update: Generative AI Content

### DISCUSSION THREAD

**Discussion by github-staff** [community/community#69006] [Announcements]
## We’ve recently updated our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) to include new guidelines around using generative AI in Community Discussions. Please review the Code of Conduct to familiarize yourself with this change. 

Since November 2022, we’ve been seeing major developments in the world of generative artificial intelligence (AI). As the world advances and changes you can expect our community to do so as well. 

We’ve taken note over the past months of how generative AI is being used to answer questions in the Community Discussions, and how community members feel about it being used. Thank you for your feedback and comments on this topic. Based on our findings we have made the following decisions to strengthen the integrity of content in the GitHub Community Discussions.

### TL;DR: While we encourage non-AI answers to questions in the community, we also love experimenting and using new technologies. If you wish to use AI generated content when answering questions in the community, you must do so by the following rules:
- Take personal responsibility for everything you post. 
- Read and revise the content before you post it; use your own authentic voice. 
- Use your expertise as a developer to verify that the answer works and makes sense.
- Do not just post AI-generated content verbatim to inflate your reputation or give a false impression of product expertise.
- AI tools will often answer in an authoritative tone that sounds like a tech support professional. Be careful not to [mislead other users](https://docs.github.com/en/site-policy/acceptable-use-policies/github-impersonation) into thinking that this authoritative tone means they are receiving an official response from GitHub. 

Additionally, all of the guidelines listed in [Contribute in a Positive Way](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct#best-practices-for-maintaining-a-strong-community) also apply here.

### Failure to adhere to these guidelines could result in the following:
- Content Removal
- Content Blocking
- Community access temporarily or permanently blocked 
- GitHub Account Suspension
- GitHub Account Termination

Our [Code of Conduct](https://docs.github.com/en/site-policy/github-terms/github-community-code-of-conduct) details further how generative AI may or may not be used in the community and how this policy will be enforced. The community is here for users to build trust through authentic reputations. We thank those who have given honest feedback, reported bad actors, and have been  patient with us as we have made these updates. We also want to  thank you for  helping us to maintain a community built on trust and authenticity. 

This post has the comments disabled as this isn’t a debate on generative AI, but you can let us know your thoughts with the handy reactions on the post.

 For conversations and questions around generative AI and large language models head over to our
 [Machine Learning and ML Ops](https://github.com/orgs/community/discussions/categories/machine-learning-and-mlops) category to start a new discussion.

**Comment by novium2** [comment]
Retiring this post as it is over 6 months old. If you have any questions, consider starting a [new post](https://github.com/orgs/community/discussions/new/choose).
